porcupine/plugins/markdown.py

```diff
@@ -0,0 +1,109 @@
+"""Features for working with Markdown Files.
+
+- Indenting and dedenting lists
+"""
+
+from __future__ import annotations
+
+import logging
+import re
+import tkinter
+from functools import partial
+
+from porcupine import get_tab_manager, tabs, textutils, utils
+
+log = logging.getLogger(__name__)
+
+
+setup_before = ["tabs2spaces", "autoindent"]
```

Comment:
Maybe must be also be before autocomplete, so that `return "break"` prevents autocompletions?

You can try running Porcupine with `--shuffle-plugins` a few times to see if you have defined all necessary dependencies. (See `--help`)

Reply:
The tests run with `--shuffle-plugins`, so shouldn't that be covered?

This actually doesn't interfere with autocomplete, the regex doesn't match useless there is a space after the list prefix, so regardless we should be good I believe. If you think that's wrong, or can think of a test to verify that, I'm all ears :)